packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -259,6 +279,7 @@ class _WrappedScrollBehavior implements ScrollBehavior {
         || oldDelegate.overscrollIndicator != overscrollIndicator
         || oldDelegate.physics != physics
         || oldDelegate.platform != platform
+        || listEquals(oldDelegate.dragDevices, dragDevices)
```

Comment:
we should probably use set otherwise the order difference may cause a rebuild.

also we should set the type? i.e. listEquals<PointerDeviceKind>